tests/test_trial.py

```diff
@@ -210,37 +210,24 @@ def test_suggest_low_equals_high(storage_init_func):
-        )  # Suggesting the same param.
-        assert mock_object.call_count == 0
-        assert trial.suggest_int("d", 1, 1) == 1  # Suggesting a param.
-        assert trial.suggest_int("d", 1, 1) == 1  # Suggesting the same param.
-        assert mock_object.call_count == 0
-        assert trial.suggest_float("e", 1.0, 1.0) == 1.0  # Suggesting a param.
-        assert trial.suggest_float("e", 1.0, 1.0) == 1.0  # Suggesting the same param.
-        assert mock_object.call_count == 0
-        assert trial.suggest_float("f", 0.5, 0.5, log=True) == 0.5  # Suggesting a param.
-        assert trial.suggest_float("f", 0.5, 0.5, log=True) == 0.5  # Suggesting the same param.
-        assert mock_object.call_count == 0
-        assert trial.suggest_float("g", 0.5, 0.5, log=False) == 0.5  # Suggesting a param.
-        assert trial.suggest_float("g", 0.5, 0.5, log=False) == 0.5  # Suggesting the same param.
-        assert mock_object.call_count == 0
-        assert trial.suggest_float("h", 0.5, 0.5, step=1.0) == 0.5  # Suggesting a param.
-        assert trial.suggest_float("h", 0.5, 0.5, step=1.0) == 0.5  # Suggesting the same param.
-        assert mock_object.call_count == 0
-        assert trial.suggest_int("i", 1, 1, log=True) == 1  # Suggesting a param.
-        assert trial.suggest_int("i", 1, 1, log=True) == 1  # Suggesting the same param.
-        assert mock_object.call_count == 0
+    assert trial.suggest_uniform("a", 1.0, 1.0) == 1.0  # Suggesting a param.
+    assert trial.suggest_uniform("a", 1.0, 1.0) == 1.0  # Suggesting the same param.
+    assert trial.suggest_loguniform("b", 1.0, 1.0) == 1.0  # Suggesting a param.
+    assert trial.suggest_loguniform("b", 1.0, 1.0) == 1.0  # Suggesting the same param.
+    assert trial.suggest_discrete_uniform("c", 1.0, 1.0, 1.0) == 1.0  # Suggesting a param.
+    assert trial.suggest_discrete_uniform("c", 1.0, 1.0, 1.0) == 1.0  # Suggesting the same param.
+    assert trial.suggest_int("d", 1, 1) == 1  # Suggesting a param.
+    assert trial.suggest_int("d", 1, 1) == 1  # Suggesting the same param.
+    assert trial.suggest_float("e", 1.0, 1.0) == 1.0  # Suggesting a param.
+    assert trial.suggest_float("e", 1.0, 1.0) == 1.0  # Suggesting the same param.
+    assert trial.suggest_float("f", 0.5, 0.5, log=True) == 0.5  # Suggesting a param.
+    assert trial.suggest_float("f", 0.5, 0.5, log=True) == 0.5  # Suggesting the same param.
+    assert trial.suggest_float("g", 0.5, 0.5, log=False) == 0.5  # Suggesting a param.
+    assert trial.suggest_float("g", 0.5, 0.5, log=False) == 0.5  # Suggesting the same param.
+    assert trial.suggest_float("h", 0.5, 0.5, step=1.0) == 0.5  # Suggesting a param.
+    assert trial.suggest_float("h", 0.5, 0.5, step=1.0) == 0.5  # Suggesting the same param.
+    assert trial.suggest_int("i", 1, 1, log=True) == 1  # Suggesting a param.
+    assert trial.suggest_int("i", 1, 1, log=True) == 1  # Suggesting the same param.
```

Comment:
Maybe we can remove this test function altogether.

Reply:
I found it useful to make sure that `sampler.sample_independent` is not called. If you don't want to add the check, I think you can remove this test function.
If you remove this test, I'm not sure if there is a test that calls the `distributions._get_single_value ()` method. I want to make sure that tests for the method are added when you remove this test.

Reply:
Right, thanks for you suggestion. I changed this test to check the number of calls to `distributions._get_single_value`. What do you think?